odl_video/envs.py

```diff
@@ -134,3 +134,22 @@ def get_any(name, default):
             return get_int(name, default)
         except EnvironmentVariableParseException:
             return get_string(name, default)
+
+
+def get_key(name, default):
+    """
+    Get an environment variable as a string representing a private or public key.
+    The difference is that keys are automatically escaped and they need to be unescaped and
+    encoded into bytestrings.
+
+     Args:
+        name (str): An environment variable name
+        default (str): The default value to use if the environment variable doesn't exist.
+
+    Returns:
+        bytes: The environment variable value, or the default as bytestring
+    """
+    value = get_string(name, default)
+    if not value:
```

Comment:
Can we check `isinstance(value, str)` instead so that the default value can be parsed if it's a string?